addons/sys_rack/fnc_activateRackSpeaker.sqf

```diff
@@ -0,0 +1,24 @@
+ #include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Sets the mounted radio external audio to ON/OFF.
```

Comment:
`* Toggles the mounted radio external audio.`
Maybe rename the function accordingly as well.